frontend/test/cypress/support/commands.ts

```diff
@@ -38,15 +38,19 @@ Cypress.Commands.add('loadProductPageByPath', (path: string) => {
    cy.wait('@user-api');
 });
 
-Cypress.Commands.add('interceptLogin', (discount_tier?: string) => {
+Cypress.Commands.add('interceptLogin', (optional?: Object) => {
+   let defaultParams = {
+      userid: 1,
+      algoliaApiKeyProduct: null,
+      username: 'john',
+      unique_username: 'john123',
+   };
+
    cy.intercept(
       { method: 'GET', url: '/api/2.0/user' },
       {
-         userid: 1,
-         algoliaApiKeyProduct: null,
-         username: 'john',
-         unique_username: 'john123',
-         discount_tier: discount_tier,
+         ...defaultParams,
+         ...optional,
```

Comment:
Will the optional params override the default ones?

Reply:
Yup, looks like the optional params will override the default:

<img width="933" alt="Screen Shot 2022-11-18 at 3 28 52 PM" src="https://user-images.githubusercontent.com/58952979/202819790-868e5097-b05e-40e6-b834-eed68325865e.png">